src/gtk/display.cpp

```diff
@@ -40,7 +158,13 @@ static inline int wx_gdk_screen_get_primary_monitor(GdkScreen* screen)
     return monitor;
 }
 #define gdk_screen_get_primary_monitor wx_gdk_screen_get_primary_monitor
-#endif // !__WXGTK3__
+
+static inline bool gdk_is_x11_screen(GdkScreen* WXUNUSED(screen))
```

Comment:
I think it would be better if we did not name things in a way that makes them look like they are part of a different library. In this case, `gdk_` prefix looks like something in GDK. I generally add some sort of `wx` prefix to make it clear.

Reply:
Renamed to `wxIsX11GDKScreen()`.

Thanks for the review!